docs/guides/high-availability/deploy-infinispan-kubernetes-crossdc.adoc

```diff
@@ -251,8 +252,48 @@ kubectl wait --for condition=WellFormed --timeout=300s infinispans.infinispan.or
 kubectl wait --for condition=CrossSiteViewFormed --timeout=300s infinispans.infinispan.org -n {ns} {cluster-name}
 ----
 
+== Connecting {jdgserver_name} with {project_name}
+[#connecting-to-keycloak]
+
+Now that an {jdgserver_name} server is running, here are the relevant {project_name} CR changes necessary to connect it to {project_name}. These changes will be required in the <@links.ha id="deploy-keycloak-kubernetes" /> {section}.
+
+. Create a Secret with the username and password to connect to the external {jdgserver_name} deployment:
++
+[source,yaml]
+----
+include::examples/generated/keycloak-ispn.yaml[tag=keycloak-ispn-secret]
+----
+
+. Extend the {project_name} Custom Resource with `additionalOptions` as shown below.
++
+[NOTE]
+====
+All the memory, resource and database configurations are skipped from the CR below as they have been described in <@links.ha id="deploy-keycloak-kubernetes" /> {section} already.
+Administrators should leave those configurations untouched.
+====
++
+[source,yaml]
+----
+include::examples/generated/keycloak-ispn.yaml[tag=keycloak-ispn]
+----
+<1> The hostname of the remote {jdgserver_name} cluster.
+<2> The port of the remote {jdgserver_name} cluster.
+This is optional and it default to `11222`.
+<3> The Secret `name` and `key` with the {jdgserver_name} username credential.
+<4> The Secret `name` and `key` with the {jdgserver_name} password credential.
+<5> The `spi-connections-infinispan-quarkus-site-name` is an arbitrary {jdgserver_name} site name which {project_name} needs for its Infinispan caches deployment when a remote store is used.
```

Comment:
I don't think we need this any more as we're no longer using a RemoteStore.

Reply:
We are still using remote store for authentication sessions, login failures and action tokens. What do you mean we are not using it?

Reply:
The RemoteCache is using the same code for initializing Infinispan connection as the old RemoteStore therefore this property still needs to be set. 

If I remember correctly, the purpose of this variable is so Keycloak can distinguish between notifications sent from this cluster and the other cluster (based on that there are some optimization to not react on some events when the originator of the event is from the same cluster as me) and since we are using the same `work` cache and same events logic I think we still need this. 

@pruivo Does it still make sense to have this property or I am wrong?

Reply:
There is only one place where the filtering by site name is used:

https://github.com/keycloak/keycloak/blob/e62604b1ec64da2981b02ba2868994f61908e6a0/model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/remote/RemoteInfinispanAuthenticationSessionProvider.java#L106-L117

Looking more closely, the method implementation seems broken. It should be a no-op. 

I'll leave the option for now.

Reply:
Thanks for details @pruivo, I will resolve this discussion in this PR, if I understand correctly at the moment we will leave everything as is but if there is some follow-up let's create a new issue for that.